packages/block-library/src/style.scss

```diff
@@ -177,7 +177,7 @@ figcaption {
 // Apply some base styles to blocks that need them.
 img {
 	max-width: 100%;
-	height: auto;
+	//height: auto;
```

Comment:
@jasmussen Is this commenting out intentional?

Reply:
Not intentional. height: auto; should remain, good catch.